apps/mail/components/mail/mail-display.tsx

```
@@ -154,6 +156,81 @@ const MailDisplay = ({ emailData, isMuted, index, totalEmails, demo }: Props) =>
+          .filter(Boolean)
+          .join(' ');
+          
+        return isBlock ? `\n${text}\n` : text;
+      }
+      
+      return '';
+    };
+
+    // Get the clean text content
+    let textContent = getCleanText(tempDiv)
+      .replace(/[\u200B-\u200D\uFEFF]/g, '') // Remove zero-width spaces
```

Biome (1.9.4)

[error] A character class cannot match a joined character sequence.

A zero width joiner composes several emojis into a new one. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)